### PR TITLE
CLN removing trailing commas

### PR DESCRIPTION
**Pull request by sarthakvk**
This PR is related to #35925, both current and new versions of black passes these formatting,
also I'd like to mention  that there seem to be a reported [bug](https://github.com/psf/black/issues/1629) in newer version of black which throws error after removing trailing comma from  `pandas/tests/scalar/timestamp/test_constructors.py` that's why I didn't change this formatting.
```diff
      diff --git a/pandas/tests/scalar/timestamp/test_constructors.py b/pandas/tests/scalar/timestamp/test_constructors.py
index 316a299ba..c70eacdfa 100644
--- a/pandas/tests/scalar/timestamp/test_constructors.py
+++ b/pandas/tests/scalar/timestamp/test_constructors.py
@@ -267,7 +267,7 @@ class TestTimestampConstructors:
                 hour=1,
                 minute=2,
                 second=3,
-                microsecond=999999,
+                microsecond=999999
             )
         ) == repr(Timestamp("2015-11-12 01:02:03.999999"))
 
~
```
The workaround to this is after removing trailing comma from above code run black with --fast which will add trailing comma back again and and will not format afterwards on re-running black without --fast

### CODE REVIEW

File: pandas/tests/io/test_sql.py [hunk 1]
```diff
@@ -2349,9 +2349,7 @@ def date_format(dt):
 
 
 def format_query(sql, *args):
-    """
-
-    """
+    """"""
```
Comment (MarcoGorelli) [inline pandas/tests/io/test_sql.py:2352]: Perhaps this docstring can be removed entirely if it's just empty anyway?

**Comment by MarcoGorelli** [comment]
Thanks for looking into this, I remember getting some errors when running the new version of black on all the files

**Review by MarcoGorelli — APPROVED** [review]
looks good to me pending green

**Comment by MarcoGorelli** [comment]
Can you merge master to get the CI green please?

**Comment by sarthakchaudhary13** [comment]
> Can you merge master to get the CI green please?

@MarcoGorelli I was wondering why CI isn't passing, I'll merge it.
Thanks

**Comment by simonjayhawkins** [comment]
Thanks @sarthakchaudhary13

**Pull request merged**